director/settings.py

```diff
@@ -445,6 +446,9 @@ class Prod(Common):
     # Note: The DSN is not a secret https://forum.sentry.io/t/dsn-private-public/6297/2
     SENTRY_DSN = values.Value('https://6329017160394100b21be92165555d72@sentry.io/37250')
 
+    # Use PostHog for product analytics
+    POSTHOG_KEY = values.Value('LeXA_J7NbIow0-mEejPwazN7WvZCj-mFKSvLL5oM4w0')
```

Comment:
What do you think about making an issue for extracting these (`POSTHOG_KEY`, `SENTRY_DSN`, plus any other similar values) into some sort of `env` file down the road. That way if someone pulls and deploys the HUB, they don't end up polluting/sending events to our account?

Reply:
Yeah, I did the same thing with Intercom, I initially put the settings in here because "they're public anyway" but had the same thoughts as Alex so moved them to the ops repo.

Reply:
I was trying to reduce the config sprawl that we currently have - config settings exist here, in k8s yaml files, and in k8s cluster secrets. But I hadn't thought about the pollution issue. So, yes please create that issue as a reminder.